test/NCalc.Tests/Fixtures.cs

```diff
@@ -112,7 +112,7 @@ public void ExpressionDoesNotDefineNullParameterWithoutNullOption()
             var e = new Expression("'a string' == null");
 
             var ex = Assert.Throws<ArgumentException>(() => e.Evaluate());
-            Assert.Contains("Parameter name: null", ex.Message);
+            Assert.Contains("Parameter was not defined", ex.Message);
```

Comment:
how come this test passed before and is still passing with the change? 🤔

Reply:
Indeed, I don't know why does it passing on AppVeyor, but it fails in Visual Studio. Seems to be an issue on AppVeyor side...

Reply:
Interesting  - sounds like something that needs to be fixed in CI